### PR TITLE
APP-4695 Use classes from UItoolkitStyles

### PR DESCRIPTION
**Pull request by symphony-annaLlorens**
### Improvement
- Use classes from UItoolkitStyles

### Demo
Screenshot 2021-11-19 at 16 14 14

### CODE REVIEW

File: stories/DropdownMenu.stories.tsx [hunk 1]
```diff
@@ -23,10 +23,10 @@ const Default: React.FC = () => (
         <Icon iconName="minus-round" className="leftIcon" />
         Cut
       </DropdownMenuItem>
-      <DropdownMenuItem className="withRightIcon">
+      <DropdownMenuItem>
         <Icon iconName="copy" className="leftIcon" />
```
Comment (symphony-adnane) [inline stories/DropdownMenu.stories.tsx:27]: is the leftIcon class needed?

File: stories/DropdownMenu.stories.tsx [hunk 2]
```diff
@@ -62,10 +62,10 @@ const WithOrWithoutIcons: React.FC = () => {
           <Icon iconName="minus-round" className="leftIcon" />
           Cut
         </DropdownMenuItem>
-        <DropdownMenuItem className="withRightIcon">
-          <Icon iconName="copy" className="leftIcon" />
+        <DropdownMenuItem>
+          <Icon iconName="copy" className="tk-dropdown-menu__icon" />
```
Comment (symphony-adnane) [inline stories/DropdownMenu.stories.tsx:66]: is it necessary to add this __icon class?

**Comment by symphony-adnane** [comment]
in your screenshot it seems there's a spacing issue between icons and test if you could fix that it'd be nice

File: stories/DropdownMenu.stories.tsx [hunk 3]
```diff
@@ -20,25 +20,25 @@ const Default: React.FC = () => (
...
+        <Icon iconName="minus-round"/>
         Cut
       </DropdownMenuItem>
-      <DropdownMenuItem className="withRightIcon">
-        <Icon iconName="copy" className="leftIcon" />
+      <DropdownMenuItem>
+        <Icon iconName="copy"/>
         Copy
-        <Icon iconName="check" className="rightIcon" />
+        <i className="tk-dropdown-menu--selected" />
```
Comment (AliciaSymphony) [inline stories/DropdownMenu.stories.tsx:29]: It'd makes more sense to have this class attached to the whole item, than just the icon

Comment (AliciaSymphony) [reply stories/DropdownMenu.stories.tsx:29]: Actually I am not sure to understand why we don't use the <Icon component, is it a specific

Comment (symphony-annaLlorens) [reply stories/DropdownMenu.stories.tsx:29]: Yes, it could be a nice improvement to have a prop selected?= true

**Review by AliciaSymphony — APPROVED** [review]

**Review by symphony-jean-michael — APPROVED** [review]

**Pull request merged**